internal/mediators/config/rules.go

```diff
@@ -26,12 +26,14 @@ type Rules map[string]Rule
 
 var rules = make(Rules)
 
-func GetRule(key string) Rule {
+// GetRule returns the rule for the given key, or the default rule if none exists.
+// Also returns whether or not a rule was found.
+func GetRule(key string) (Rule, bool) {
 	rule, ok := rules[key]
 	if !ok {
-		return defaultRule
+		return defaultRule, false
 	}
-	return rule
+	return rule, true
```

Comment:
I don't think this is adding anything useful. Now we need to check the bool and have specialized knowledge in the consumer. The previous logic already covered this use case by not actually doing anything with the rule other than satisfying the expectations of the returned type.

Reply:
Okay.